src/pages/index.js

```diff
@@ -0,0 +1,333 @@
+    handleDeleteCard(cardElement, data._id);
+  });
+
+  return cardElement;
+}
+
+cancelButton.addEventListener("click", (evt) => {
+  closeModal(deleteModal);
+});
+
+function handleDeleteSubmit(evt) {
+  evt.preventDefault();
+  const deleteButton = evt.submitter;
+  setButtonText({
+    btn: deleteButton,
+    isLoading: true,
+    defaultText: "Delete",
+    loadingText: "Deleting...",
+  });
+  api
+    .deleteCard(selectedCardId)
+    .then(() => {
+      selectedCard.remove();
+      closeModal(deleteModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText({
+        btn: deleteButton,
+        isLoading: false,
+        defaultText: "Delete",
+        loadingText: "Deleting...",
+      });
+    });
+}
+
+function handleDeleteCard(cardElement, cardId) {
+  selectedCard = cardElement;
+  selectedCardId = cardId;
+  openModal(deleteModal);
+}
+
+function handlePressEsc(evt) {
+  if (evt.key === "Escape") {
+    const modal = document.querySelector(".modal_opened");
+    if (modal) {
+      closeModal(modal);
+    }
+  }
+}
+
+function openModal(modal) {
+  document.addEventListener("keydown", handlePressEsc);
+  modal.classList.add("modal_opened");
+}
+
+function closeModal(modal) {
+  document.removeEventListener("keydown", handlePressEsc);
+  modal.classList.remove("modal_opened");
+}
+
+function renderCard(cardData, placement = "prepend") {
+  const cardElement = getCardElement(cardData);
+
+  if (placement === "append") {
+    cardsList.append(cardElement);
+  } else {
+    cardsList.prepend(cardElement);
+  }
+}
+
+function handleAddCardSubmit(evt) {
+  evt.preventDefault();
+  const inputValues = { name: cardNameInput.value, link: cardLinkInput.value };
+  const modal__submit = evt.submitter;
+  setButtonText({ btn: modal__submit, isLoading: true });
+  api
+    .addCard(inputValues)
+    .then((cardData) => {
+      renderCard(cardData, "prepend");
+      closeModal(cardModal);
+      disableButton(cardSubmitBtn, settings);
+      cardFormElement.reset();
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText({ btn: modal__submit, isLoading: false });
+    });
+}
+
+function handleEditFormSubmit(evt) {
+  evt.preventDefault();
+  const modal__submit = evt.submitter;
+  setButtonText({
+    btn: modal__submit,
+    isLoading: true,
+    defaultText: "Save",
+    loadingText: "Saving...",
+  });
+
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      jobElement.textContent = data.about;
+      closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText({
+        btn: modal__submit,
+        isLoading: false,
+        defaultText: "Save",
+        loadingText: "Saving...",
+      });
+    });
+}
+
+function handleAvatarSubmit(evt) {
+  evt.preventDefault();
+  const modal__submit = evt.submitter;
```

Comment:
`modal__submit`

Every variable in the project should be in `camelCase` 


`submitButton`